interfaces/policy/policy.go

```diff
@@ -155,6 +155,10 @@ func (connc *ConnectCandidate) SlotAttr(arg string) (interface{}, error) {
 	return nestedGet("slot", connc.Slot, arg)
 }
 
+func (connc *ConnectCandidate) PlugPublisherID(_ string) (interface{}, error) {
```

Comment:
Perhaps a comment that `PLUG_PUBLISHER_ID` doesn't take an argument, hence it's not used. And maybe throw an error if the given string is not empty?

Reply:
I'll just create a new matcher just for references that use this. That way this doesn't have to match a sort-of unrelated interface and it also solves the comment below. I thought it might be overkill since the new functionality is so similar but it might be cleaner